### ISSUE TITLE
html-closing-bracket-newline is being ignored

### ISSUE THREAD

**Issue by MattRynbrand** [vuejs/eslint-plugin-vue#859]
**Tell us about your environment**	
* **ESLint version: 5.15.1**	
* **eslint-plugin-vue version: 5.2.2**	
* **Node version:**	

**Please show your full configuration:**

```json
module.exports = {
    plugins: [
        'flowtype',
        'vue',
    ],
    extends: [
        'plugin:vue/recommended',
        'eslint:recommended',
    ],
    rules: {
        // Generic Javascript Rules
        'space-before-blocks': [2, 'always'],
        semi: 'error',
        quotes: [
            1,
            'single',
            {
                allowTemplateLiterals: true
            }
        ],
        'comma-dangle': ['warn', 'never'],
        'no-console': 0,
        'new-cap': [
            'error',
            {
                newIsCap: true,
                properties: true,
                capIsNewExceptions: [
                    'Definition',
                    'Model',
                    'Requery',
                    'QueryAggregator',
                    'GqlFetcher',
                    'Mutator',
                    'Query',
                    'QueryAssembler',
                    'Listener',
                    'Paginator',
                    'Faker',
                    'Sorter',
                    'ModuleHelper',
                    'User'
                ]
            }
        ],
        'no-multiple-empty-lines': [
            'error',
            {
                max: 1,
                maxEOF: 1
            }
        ],
        'arrow-parens': ['error', 'as-needed'],
        'no-var': 'error',
        'key-spacing': [
            'error',
            {
                afterColon: true
            }
        ],
        'object-curly-newline': [
            'error',
            {
                minProperties: 2,
                consistent: true
            }
        ],
        'object-curly-spacing': ['error', 'always'],
        'brace-style': ['error', 'stroustrup', {
            'allowSingleLine': true
        }],
        indent: [
            'error',
            4,
            {
                SwitchCase: 1,
                VariableDeclarator: 1
            }
        ],

        // Vue Rules
        'vue/prop-name-casing': 0,
        'vue/html-self-closing': 0,
        'vue/html-indent': ['error', 4, {
            attribute: 1,
            baseIndent: 1,
            closeBracket: 0,
            alignAttributesVertically: true
        }],
        'vue/html-closing-bracket-spacing': ['error', {
            'startTag': 'never',
            'endTag': 'never',
            'selfClosingTag': 'never'
        }],
        'vue/script-indent': ['error', 4, {
            'baseIndent': 1
        }],

        "vue/html-closing-bracket-newline": ["error", {
            "singleline": "never",
            "multiline": "never"
        }]

        // Flow Rules
        'flowtype/boolean-style': [2, 'boolean'],
        'flowtype/define-flow-type': 1,
        'flowtype/delimiter-dangle': [2, 'never'],
        'flowtype/generic-spacing': [2, 'never'],
        'flowtype/no-primitive-constructor-types': 2,
        'flowtype/no-types-missing-file-annotation': 0,
        'flowtype/object-type-delimiter': [2, 'comma'],
        'flowtype/require-valid-file-annotation': 2,
        'flowtype/semi': [2, 'always'],
        'flowtype/space-after-type-colon': [2, 'always'],
        'flowtype/space-before-generic-bracket': [2, 'never'],
        'flowtype/space-before-type-colon': [2, 'never'],
        'flowtype/type-id-match': [2, '^([A-Z][a-z0-9]+)+Type$'],
        'flowtype/union-intersection-spacing': [2, 'always'],
        'flowtype/use-flow-type': 1,
        'flowtype/valid-syntax': 1,
        'flowtype/require-parameter-type': [2, {
            excludeArrowFunctions: true
        }],
        'flowtype/require-return-type': [2, 'always', {
            annotateUndefined: 'never',
            excludeArrowFunctions: true
        }]
    },
    settings: {
        flowtype: {
            onlyFilesWithFlowAnnotation: false
        }
    },

    parserOptions: {
        parser: 'babel-eslint',
        ecmaVersion: 6,
        sourceType: 'module'
    },
    globals: {
        // Node Globals
        __dirname: 'readonly',
        process: 'readonly',

        // Misc libraries
        $: 'readonly',
        _: 'readonly',
        module: 'readonly',
        is: 'readonly',

        // Services
        Trans: 'readonly',
        Validator: 'readonly',
        Config: 'readonly',
        require: 'readonly',
        Make: 'readonly',
        DataTypes: 'readonly',
        Entities: 'readonly',
        Enums: 'readonly',
        Utils: 'readonly',
        Events: 'readonly',
        Me: 'readonly',
        Vue: 'readonly',
        Formatter: 'readonly',
        SC: 'readonly',
        MeService: 'readonly',
        Reference: 'readonly',
        FormHelper: 'readonly',
        Url: 'readonly',

        // Tests
        describe: 'readonly',
        it: 'readonly'
    },

    'overrides': [{
        'files': ['*.vue'],
        'rules': {
            'indent': 'off'
        }
    }]
};
```

**What did you do?**	

```vue
<template>
	<div>
		<test-component
			test-attr1="123"
			v-for="i in array"
			:key="i"
			class="test"
			@click="test()"
		>when content gets put here it puts the closing tag on a new line</test-component>
	</div>
</template>
```

**What did you expect to happen?**
Not to put `>` characters on a new line

**What actually happened?**

ESLint doesn't output an error but still reformats is as if there is an error
